BackendWebApp/src/main/java/uk/ac/ebi/biostd/webapp/application/validation/eutoxrisk/services/EUToxRiskFileValidator.java

```diff
@@ -0,0 +1,41 @@
+package uk.ac.ebi.biostd.webapp.application.validation.eutoxrisk.services;
+
+import org.springframework.core.io.FileSystemResource;
+import org.springframework.http.HttpEntity;
+import org.springframework.http.HttpHeaders;
+import org.springframework.http.MediaType;
+import org.springframework.http.converter.json.MappingJackson2HttpMessageConverter;
+import org.springframework.web.client.RestTemplate;
+import uk.ac.ebi.biostd.webapp.application.validation.eutoxrisk.dto.EUToxRiskFileValidationError;
+import uk.ac.ebi.biostd.webapp.application.validation.eutoxrisk.dto.EUToxRiskFileValidationResponse;
+
+import java.io.File;
+import java.util.Collection;
+
+/**
+ * @author Olga Melnichuk
+ */
+public class EUToxRiskFileValidator {
+
+    private final RestTemplate restTemplate;
+    private final String url;
+
+    public EUToxRiskFileValidator(RestTemplate restTemplate,
```

Comment:
to follow google convention standart this should be 

```
public EUToxRiskFileValidator(RestTemplate restTemplate,  String url) {
```

Reply:
:-)) done!